docs/content/stable/explore/transactions/explicit-locking.md

```diff
@@ -110,69 +114,7 @@ In PostgreSQL, if an advisory lock is taken on one session, all sessions should
 
 Advisory locks are enabled by default. You can configure advisory locks using the [Advisory lock flags](../../../reference/configuration/yb-tserver/#advisory-lock-flags).
```

Comment:
Change the advisory lock link in tserver reference to link to architecture instead of explore